### ISSUE TITLE
20180310

### ISSUE THREAD

**Issue by HurleyJames** [HurleyJames/IELTS_Writing#6]
Some people think that one of the best ways to solve environment problems is to increase the cost of the fuel for cars and other vehicles. To what extent do you agree or disagree? 

Some people argue that increasing the price for fuel is one of the best solutions to environment problems, but I cannot agree with this opinion. 
To begin with, education is can raise citizen's environmental awareness. After receiving education about ecology, individuals may understand how important environmental protection is and what severe pollution the earth is suffering. Then, they are more likely to protect the environment actively in daily life, such as classifying rubbish. Such everyday activities can effectively reduce the amount of rubbish produced by human activities, as well as the adverse impact on the environment. 
In addition, environmental problems may not be solved without law. If governments can enforce laws and regulations to protect the environment, more citizens will choose to comply with laws and stop their behavior that may damage the environment. For example, if citizens face a heavy fine after they spit casually on the street, most of them will change such a habit, and the environment can be clearer. Therefore, law is necessary for environmental protection because it forces people to behave properly.
Admittedly, some people argue that increasing the price for fuel can reduce the use of private vehicles, and the environment can be protected since less toxic gas will be emitted by private cars. However, they are overoptimistic about this method. The fact is that, since people can afford to purchase a car, it means that they can also afford the price for fuel. Even if the price for fuel increases, most drivers may just complain about the increasing price for fuel, but it does not prevent them from driving their own cars, and the quality of environmnet may not be improved significantly. 
To sum up, increasing the price for fuel may not be a useful solution to environmental problems. Instead, education and law are two necessary methods that can reduce pollution effectively.
